test/api_client_tests/test_event.py

```diff
@@ -430,7 +430,7 @@ def test_event_sp(sts):
         e
         for e in events
         if e.type_ == "no.elhub.flex.technical_resource.create"
-        and e.source == f"/technical_resource/{tr.id}"
+        and e.subject == f"/technical_resource/{tr.id}"
```

Comment:
Should we assert source in these tests aswell? Or is it covered by another test?